logfire/_internal/main.py

```diff
@@ -1863,6 +1871,10 @@ def set_attributes(self, attributes: dict[str, Any]) -> None:
         for key, value in attributes.items():
             self.set_attribute(key, value)
 
+    def add_link(self, context: SpanContext, attributes: otel_types.Attributes = None) -> None:
+        if self._span is not None:  # pragma: no branch
```

Comment:
the other branch should be handled, i.e. the link should be stored somewhere in case the span is started later